packages/spark-core/utilities/polyfills/focusWithin.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
+(function(window, document){
```

Comment:
Should we convert this to ES6 to match the rest of our JS?

Reply:
ended up removing the polyfill